src/index.ts

```diff
@@ -1,4 +1,8 @@
-import { NativeModules, NativeEventEmitter, EmitterSubscription } from 'react-native';
+import {
```

Comment:
hard to find what had changed aside from prettier. can you add comments to make them stand out?

Reply:
The only change is the changes in `android/src/main/java/com/amplitude/reactnative/AmplitudeReactNativeModule.java`